### ISSUE TITLE
Three.js canvas does not resize properly

### ISSUE THREAD

**Issue by nfoert** [nfoert/circles#4]
When the window is resized or the console is opened the elements in the canvas are squashed instead of resized. There is already code adressing this but it does not seem to working as expected.

In `main.js`:
```py
window.addEventListener('resize', onWindowResize, false);

function onWindowResize() {
    camera.aspect = window.innerWidth / window.innerHeight;
    camera.aspect = window.innerWidth / window.innerHeight;
    camera.updateProjectionMatrix();
    
    renderer.setSize(window.innerWidth, window.innerHeight);

    renderer.render(scene, camera);

}
```

**Comment by charann29** [comment]
@nfoert  I like to contribute to this issue please assign me this issue

**Comment by nfoert** [comment]
> @nfoert I like to contribute to this issue please assign me this issue

How about you do the first issue I assigned you on first, and then if no one else has claimed this one I can assign you too.
These are relatively simple issues and I would hope for some other people to learn about contributing as well.

**Comment by charann29** [comment]
Okay @nfoert , No issues

**Comment by Pratikdate** [comment]
Assign me i will work upon it

**Comment by nfoert** [comment]
> Assign me i will work upon it

Of course! Thanks for contributing.

**Comment by Pratikdate** [comment]
Thanks

**Comment by nfoert** [comment]
Hey @Pratikdate! Do you have any updates on this issue?

**Comment by Pratikdate** [comment]
No

**Comment by nfoert** [comment]
@Pratikdate, do you still plan on working on this issue? If not I can give it to @bilalmirza74.

**Comment by Pratikdate** [comment]
no

**Comment by nfoert** [comment]
Ok @bilalmirza74, It's all yours. 

This is what the canvas should look like:
![image](https://github.com/nfoert/circles/assets/92690073/7f080ed4-fd96-4259-a526-688b21ffe672)

When you open the console or resize the window, the canvas becomes stretched:
![image](https://github.com/nfoert/circles/assets/92690073/d3d28c20-3e81-48ac-98e4-7481c9afd4c6)

This should be fixed at this line, (and the `onWindowResize` function right below it) but it seems to not be working correctly.
https://github.com/nfoert/circles/blob/7f9cbbd7e5b6d66438746995274279a250af7952/circles/main/static/main/scripts/main.js#L66

Thanks!

**Comment by nfoert** [comment]
> ![image](https://user-images.githubusercontent.com/84387676/276598451-1aa2cba9-7b2f-4fbc-9cc4-63bbd4210ff8.png)
> 
> on home page

Don't forget to create a super user, log into the admin pane a create a new `Server` object, with the ip address `http://127.0.0.1:8000` and set production to `False`

**Comment by nfoert** [comment]
@bilalmirza74, are you talking about these?
![image](https://github.com/nfoert/circles/assets/92690073/79c0a38d-91c0-4ee6-8770-7aa0800100aa)

Circles allow Users to move to different spaces inside the Server. You can create them in the admin interface or in the Add menu.

It looks like you're getting a error though, when the client says "Failed to connect", what is the error in the django console?

**Comment by nfoert** [comment]
@bilalmirza74, that's pretty strange. Any chance you forgot to run `python manage.py migrate`? If you did and the error is still occurring, can you please tell me what steps I need to do to reproduce the error?

**Comment by nfoert** [comment]
Hey @bilalmirza74! Did you end up able to resolve the error?

**Comment by nfoert** [comment]
Hi, I resolved the error in #36. I don't know if you were still planning on completing it @bilalmirza74, but it had been a few weeks and I needed to resolve it. There's plenty more issues if you'd still like to contribute. Thanks!

**Comment by nfoert** [comment]
> Let me run this again on my local machine, If I won't get that wierd error I will do it definitely

Do note that I closed this issue, so the bug with the three.js canvas is no longer present. (There are a few strange behaviors when using the mobile inspector in Chromium, this doesn't seem to occur in Firefox so I'd presume that's a bug with the browser. Feel free to work on that if you have a fix for it though)

Otherwise, this bug is closed, but I'd love to see some contributions with some of the other issues!
